inc/compat/scan.php

```diff
@@ -124,7 +124,7 @@ protected function evaluate() {
 			$this->result = 'unsupported-php';
 		}
 
-		if ( \is_multisite() ) {
+		if ( \is_multisite() && 'local' !== \wp_get_environment_type() ) {
```

Comment:
Would it be worth also allowing `development` to run MS? Or, perhaps this is looking at it backwards and we should instead have an `in_array()` check that looks for `in_array( \wp_get_environment_type(), array( 'production', 'staging' ), true )` since someone _could_ define a type other than the defaults, and I believe we currently only expect these two values in our environments.